src/App.js

```diff
@@ -1,13 +1,76 @@
 import React from 'react';
+import TaskList from './components/TodoComponents/TodoList';
+import TaskForm from './components/TodoComponents/TodoForm';
+import './app.css';
+
+const toDoData = [
+];
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+
+  constructor() {
+    super();
+    this.state = {
+      taskList: toDoData,
```

Comment:
You can simply set-up the taskList on state to have an empty array `[]` instead of declaring it separately outside of state